frontend/package.json

```diff
@@ -1,37 +1,42 @@
 {
-  "name": "frontend",
-  "version": "1.0.0",
-  "description": "",
-  "main": "App.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "dev": "webpack --mode development src/App.js --output ./static/build/main.js",
```

Comment:
I don't understand what is happening here. Why are you overriding the frontend code?

Reply:
when we use the create react app, it auto generate this file...